heron/statemgrs/src/java/com/twitter/heron/statemgr/zookeeper/curator/CuratorStateManager.java

```diff
@@ -79,8 +79,8 @@ public void initialize(Config newConfig) {
 
       String newConnectionString = tunneledResults.first;
       if (newConnectionString.isEmpty()) {
-        throw new IllegalArgumentException("Cannot connect to tunnelHost: "
-            + tunnelConfig.getTunnelHost() + " Bad connectionString: " + connectionString);
+        throw new IllegalArgumentException("Cannot connect to tunnel host: "
+            + tunnelConfig.getTunnelHost());
```

Comment:
For all of these exception messages, assume they get caught at the highest level, up in the python main and rendered to the user without any of the surrounding context that we have when reading the code.

Do they contain enough context to help the user fully understand what failed and why? So when we say "Can not connect to tunnel host foo" is there any other info they might need? Maybe something about who/what/why something is trying to connect to a tunnel host in the first place (i.e. we're doing this for ZK in this case), what username are we using, etc.